README.md

```diff
@@ -22,6 +22,15 @@ The `PrivateAssets` metadata is needed to prevent `dotnet pack` or `msbuild /t:p
 
 ## Using XliffTasks
 
+### Supported file types
+
+`XliffTasks` currently supports the following source file types:
+
+- .resx
+- .vsct
+- .xaml
```

Comment:
Should probably clarify that this is not arbitrary xaml, but specifically for CPS rules.

Reply:
Ah, good call.